src/pages/patientView/PatientViewPageUtils.tsx

```diff
@@ -51,3 +70,58 @@ export function getSamplesProfiledStatus(
         notProfiledIds,
     };
 }
+export function retrieveMutationalSignatureMap(
+    inputMetaData: GenericAssayMeta[]
+): MutationalSignatureLabelMap[] {
+    const mappedData = inputMetaData.map((metaData: GenericAssayMeta) => {
+        const nameSig: string =
+            'MUTATION_TYPE' in metaData.genericEntityMetaProperties
+                ? metaData.genericEntityMetaProperties['MUTATION_TYPE']
+                : '';
+        const classSig: string =
+            'MUTATION_CLASS' in metaData.genericEntityMetaProperties
+                ? metaData.genericEntityMetaProperties['MUTATION_CLASS']
+                : '';
+        const mutNameSig: string =
+            'NAME' in metaData.genericEntityMetaProperties
+                ? metaData.genericEntityMetaProperties['NAME']
+                : '';
+        const signatureId = metaData.stableId;
+        return {
+            stableId: signatureId,
+            signatureLabel: nameSig,
+            signatureClass: classSig,
+            name: mutNameSig,
+        };
+    });
+    return mappedData;
+}
+
+export function createMutationalCountsObject(
+    inputData: any,
+    signatureLabelMap: MutationalSignatureLabelMap[]
+) {
+    let result = [];
+    if (inputData && inputData.length > 0) {
+        for (const count of inputData) {
+            let mutationalSignatureChartData: IMutationalCounts = {} as IMutationalCounts;
+            mutationalSignatureChartData.patientId = count.patientId;
+            mutationalSignatureChartData.sampleId = count.sampleId;
+            mutationalSignatureChartData.studyId = count.studyId;
+            mutationalSignatureChartData.uniquePatientKey =
+                count.uniquePatientKey;
+            mutationalSignatureChartData.uniqueSampleKey =
+                count.uniqueSampleKey;
+            mutationalSignatureChartData.version = getMutationalSignaturesVersionFromProfileId(
+                count.molecularProfileId
+            );
+            mutationalSignatureChartData.value = parseFloat(count.value);
+            mutationalSignatureChartData.mutationalSignatureLabel =
+                signatureLabelMap
+                    .filter(obj => obj.stableId === count.stableId)
+                    .map(obj => obj.name)[0] || '';
+            result.push(mutationalSignatureChartData);
+        }
```

Comment:
I would use `map` here, something like this:

```suggestion
        result = inputData.map(count => ({
                patientId: count.patientId,
                sampleId: count.sampleId,
                studyId: count.studyId,
                uniquePatientKey: count.uniquePatientKey,
                uniqueSampleKey: uniqueSampleKey,
                version: getMutationalSignaturesVersionFromProfileId(
                    count.molecularProfileId
                ),
                value: parseFloat(count.value),
                mutationalSignatureLabel: signatureLabelMap
                    .filter(obj => obj.stableId === count.stableId)
                    .map(obj => obj.name)[0] || '',
            })
        );
```

Reply:
Implemented the map function and created the type `MutationalSignatureCount` so that `count` is not of type `any`